src/content.ts

```diff
@@ -42,7 +43,10 @@ export const broadcast = async (
   const currentFromId = config.requireGetCurrentFromId(opts);
 
   const contentHash = keccak256(content);
-  const uri = await store.put(contentHash, content, opts);
```

Comment:
is contentHash a good enough file name? Could we run into problems when sending the same note. At scale, even a timestamp might not be enough to differentiate it. Perhaps, UUID might be a good idea.

Reply:
I think content hash is. If we send the same note twice with the same contents, I think it is ok if they use the same file.